### ISSUE TITLE
关于一词多义问题

### ISSUE THREAD

**Issue by OhYee** [mmjang/ankihelper#10]
如果一个单词有多个含义，对应的字典会返回多个词条。
而显然将这些词条合成一个卡片更有助于背单词。
希望在生成卡片的时候能够多选多个词条。

**Comment by bojiang** [comment]
不同意。反而是现在的设计才是最好的。制作卡片是基于阅读上下文的，很多时候我们也只是一个释义不会，一词多义把其它释义放在同一张卡片，上下文就失去了意义。另外你可以用“所有释义”来达到你的需求。
